### PR TITLE
WA-116: Adjusted create screens

### PR DESCRIPTION
**Pull request by rmeissner**
Closes WA-116.

Changes proposed in this pull request:
- Adjusted create screens

@gnosis/mobile-devs

### CODE REVIEW

**Comment by fmrsabino** [comment]
Missing:

- QRCode Scanning
- "Use Max Balance" Button

File: app/src/main/res/layout/layout_transaction_details_asset_transfer.xml [hunk 1]
```diff
@@ -105,84 +103,51 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="@dimen/transaction_details_cell_height">
+            android:drawablePadding="8dp"
+            android:hint="0"
+            android:inputType="numberDecimal"
+            android:paddingLeft="0dp"
+            android:paddingRight="0dp"
+            android:textColor="@color/light_text"
+            android:textSize="24sp"
+            tools:text="1294dfhdg83" />
```
Comment (fmrsabino) [inline app/src/main/res/layout/layout_transaction_details_asset_transfer.xml]: 🤔

Comment (rmeissner) [reply app/src/main/res/layout/layout_transaction_details_asset_transfer.xml]: Don't you know what it means?

Comment (fmrsabino) [reply app/src/main/res/layout/layout_transaction_details_asset_transfer.xml]: I don't think so 🤔

Comment (rmeissner) [reply app/src/main/res/layout/layout_transaction_details_asset_transfer.xml]: me neither :D

**Pull request merged**